boot/bootutil/src/bootutil_priv.h

```diff
@@ -58,10 +58,16 @@ struct flash_area;
      defined(MCUBOOT_SWAP_USING_MOVE) + \
      defined(MCUBOOT_DIRECT_XIP) + \
      defined(MCUBOOT_RAM_LOAD) + \
-     defined(MCUBOOT_FIRMWARE_LOADER)) > 1
+     defined(MCUBOOT_FIRMWARE_LOADER) + \
```

Comment:
I cannot find any reference to 'MCUBOOT_FIRMWARE_LOADER' in the documentation. Is it still under review / in progress?

Reply:
Like single loader, it is a zephyr-only feature, described on https://github.com/mcu-tools/mcuboot/blob/main/boot/zephyr/Kconfig#L260